### ISSUE TITLE
add orthography profile

### ISSUE THREAD

**Issue by LinguList** [lexibank/csd#9]
Before we can do this in full, we should check how much we actually get. Thus running `lingpy profile` on the data.

**Comment by LinguList** [comment]
[wiki says](https://en.wikipedia.org/wiki/Siouan_languages):

> In Siouanist literature (e.g., Rankin et al. 2015), Americanist phonetic transcriptions are the norm, so IPA *ʃ is Americanist *š, IPA *j is Americanist *y, and so on.

**Comment by LinguList** [comment]
@tresoldi, I think we need a profile here that takes context into account, just to be on the save side. I could have a first look, but would like to ask you to double check later?

**Comment by xrotwang** [comment]
If this succeeds, it sounds like a candidate for addition to `orthograpy`.

**Comment by LinguList** [comment]
[orthography-auto.txt](https://github.com/lexibank/csd/files/3904199/orthography-auto.txt)

**Comment by LinguList** [comment]
more than 2000 lines. But the reason is the form spec, which is not sufficient. There are these symbols for variants, like `~`, etc., which do not make much sense. I am inclined to say: formspec should only take the first match. It is impossible to read the difference between the many delimiters.

**Comment by tresoldi** [comment]
Oh yes, it is the same issue with the profile for dravlex, FormSpec needs to be fine-tuned before anything. Should I take care of that?

**Comment by tresoldi** [comment]
Also related to https://github.com/lexibank/csd/issues/8

**Comment by LinguList** [comment]
Yes, @tresoldi, if you have time, give it a try. As to dravlex: you HAVE seen the zip-folder with the data and a complete profile, that I put on github yesterday, right? Just to avoid that we duplicate work, as dravlex is solved, just needs to be integrated from my lingpy script.

**Comment by tresoldi** [comment]
I saw it, but I am waiting for Simon's PR to be merged first (https://github.com/lexibank/dravlex/pull/2)

**Comment by SimonGreenhill** [comment]
I'll integrate the drav lex stuff shortly
